From cd76dc1f2f0585ad3632c8319786e2c218a3f0e2 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 8 Jan 2014 16:04:43 +0800
Subject: [PATCH 285/602] ENGR00294178 config:update uevent config for new
 sensor driver

Update uevent config for new sensor version which refers to patches from
sensor team.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index a8dfd9c..3efd975 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -21,6 +21,7 @@
 /dev/graphics/fb0         0660   system     graphics
 /dev/watchdog             0660   system     root
 /dev/ir                   0660   system     system
+/dev/mma8x5x              0640   compass     system
 
 # sysfs properties
 /sys/devices/virtual/input/input*   name        0660  root   input
@@ -29,6 +30,8 @@
 /sys/devices/virtual/input/input*   poll        0660  root   input
 /sys/devices/virtual/input/input*   enable      0660  root   input
 /sys/devices/virtual/input/input*   position    0660  root   input
+/sys/devices/virtual/input/input*   poll_delay  0660  root   input
+/sys/devices/virtual/input/input*   fifo        0660  root   input
 /sys/devices/virtual/thermal/thermal_zone* trip_point_0_temp 0660 system system
 /sys/devices/virtual/thermal/thermal_zone* trip_point_1_temp 0660 system system
 /sys/devices/virtual/thermal/thermal_zone* trip_point_2_temp 0660 system system
-- 
1.8.0

